apps/omg_status/lib/omg_status/release_tasks/set_logger.ex

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019-2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Status.ReleaseTasks.SetLogger do
+  @moduledoc false
+  use Distillery.Releases.Config.Provider
+  @app :logger
+  require Logger
+
+  @impl Provider
+  def init(_) do
+    logger_backends = Application.get_env(@app, :backends, persistent: true)
+    logger_backend = get_logger_backend()
+
+    remove =
+      case logger_backend do
+        :console -> Ink
+        _ -> :console
+      end
+
+    backends = logger_backends |> Kernel.--([remove]) |> Enum.concat([logger_backend]) |> Enum.uniq()
+    :ok = Application.put_env(@app, :backends, backends, persistent: true)
+  end
+
+  defp get_logger_backend() do
+    logger =
+      "LOGGER_BACKEND"
+      |> get_env()
+      |> validate_string(Ink)
+
+    _ = Logger.info("CONFIGURATION: App: #{@app} Key: LOGGER_BACKEND Value: #{inspect(logger)}.")
+    logger
+  end
+
+  defp get_env(key), do: System.get_env(key)
+
+  defp validate_string(nil, default), do: default
+  defp validate_string(value, default), do: do_validate_string(String.upcase(value), default)
+  defp do_validate_string("CONSOLE", _default), do: :console
+  defp do_validate_string("INK", default), do: default
```

Comment:
```suggestion
  defp do_validate_string("INK", _default), do: Ink
```

So it doesn't rely on the default being `Ink` to be correct.

Reply:
If at some point we change to `@default_backend NotInk`, one needs to realize this line needs to change too, which I think is quite hard to spot. Otherwise `LOGGER_BACKEND=INK` would use `NotInk`.

And this same argument would also apply even if we didn't turn to module attr and still use `|> validate_string(Ink)`.